### ISSUE TITLE
User should be able to get optional guided tour in a `PageElement`.

### ISSUE THREAD

**Issue by anders-kiaer** [Statoil/webviz#106]
The user should have the option of getting a "guided tour" around a PageElement. 

Especially for the more complex, specialized visualizations, this is a useful way of giving more in-depth explanations not practical to give through title and labels/legends.

Several libraries exist to facilitate this, e.g. the MIT licensed library http://bootstraptour.com/.

![image](https://user-images.githubusercontent.com/31612826/45966228-d0db6a00-c02a-11e8-8a57-38340186f646.png)

**Comment by anders-kiaer** [comment]
@eivindjahren Ref. status meeting this week, here are some suggested steps for the `Map` which could perhaps be added as default tour provided by that `PageElement` :slightly_smiling_face: 

**Note:** As an extra detail, the steps in a "standard tour" will depend somewhat on the data given. When we extend the map to have an horizontal slider (e.g. time axis), and the input data inludes that dimension, that should also be part of the standard tour. But I wouldd guess this is not a big issue as long as `Map`, which knows the input data, itself provides the "standard tour".

```python
[{
  'element': '#svg_map',
  'title': 'Map',
  'placement': 'top',
  'content': ('The map can be moved and zoomed the way you usually interact with '
              'web maps.<br><br>Click&nbsp;&amp;&nbsp;drag to move the map, and '
              'roll the mouse wheel (or do the two finger pinch gesture if on a '
              'mobile device) to zoom.')
}, {
  'element': '#svg_controls > #g_scale',
  'title': 'Map scale',
  'placement': 'bottom',
  'content': 'The map scale will change according to the level of zoom you choose.'
}, {
  'element': '#svg_controls > #g_compass',
  'title': 'Map compass',
  'placement': 'left',
  'content': ('In addition to show the current map orientation, you can click '
              'and drag the compass in order to rotate the map.')
}, {
  'element': '#svg_controls > #g_slider',
  'title': 'Layer axis',
  'placement': 'auto',
  'content': 'Scroll through the different layers using the vertical slider.'
}]
```

**Comment by eivindjahren** [comment]
The functionality is now implemented in #123, but there are no default tours for any page elements. We will look at default tours for some of our visualizations in the future.

**Comment by anders-kiaer** [comment]
Moved the remaning part of this issue to #133.
